crates/uv-python/src/discovery.rs

```diff
@@ -1534,6 +1554,13 @@ impl VersionRequest {
                     ));
                 }
             }
+            Self::MajorMinorPrerelease(major, minor, _) => {
+                if (*major, *minor) < (3, 7) {
+                    return Err(format!(
+                        "Python <3.7 is not supported but {major}.{minor} was requested."
```

Comment:
Should we show the pre-release as part of the "was requested" bit here?

Reply:
Absolutely!